commons/src/main/kotlin/com/simplemobiletools/commons/dialogs/ColorPickerDialog.kt

```diff
@@ -248,8 +272,263 @@ class ColorPickerDialog(
         viewTarget.y = viewSatVal.top + y - viewTarget.height / 2
     }
 
-    private fun getColor() = Color.HSVToColor(currentColorHsv)
+    private fun getColor() = android.graphics.Color.HSVToColor(currentColorHsv)
     private fun getHue() = currentColorHsv[0]
     private fun getSat() = currentColorHsv[1]
     private fun getVal() = currentColorHsv[2]
 }
+
+@Composable
+fun ColorPickerAlertDialog(
+    alertDialogState: AlertDialogState,
+    modifier: Modifier = Modifier,
+    @ColorInt
+    color: Int,
+    recentColors: List<Int>,
+    removeDimmedBackground: Boolean = false,
+    addDefaultColorButton: Boolean = false,
+    onActiveColorChange: ((color: Int) -> Unit)? = null,
+    onButtonPressed: (wasPositivePressed: Boolean, color: Int) -> Unit
+) {
+    val view = LocalView.current
+    val context = LocalContext.current
+    var wasDimmedBackgroundRemoved by remember { mutableStateOf(false) }
+
+    AlertDialog(
+        modifier = modifier
+            .dialogBorder,
+        onDismissRequest = alertDialogState::hide,
+        properties = DialogProperties(usePlatformDefaultWidth = false)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth(0.95f)
```

Comment:
This doesn't work on Samsung devices :(

https://issuetracker.google.com/issues/260755409

Reply:
Oh, I just copied that from `LineColorPickerDialog` to ensure same behavior. I will take a look.

Reply:
I can test it on my S22 U, will attach screenshot in a bit.

Reply:
Great, thanks!

Reply:
It seems to be fixed using OneUI +5.0

Reply:
Yeah, I saw that in the thread you sent. Is this okay to keep then?

Reply:
It's best if @tibbi tests it too on his Samsung device, if it works let's keep it, this is a good sign.